### DISCUSSION TITLE
ストックの登録画面にインクリメンタルサーチができる検索窓がほしい。

### DISCUSSION THREAD

**Discussion by Teruichi81** [increments/qiita-discussions#435] [要望・アイデア・不具合]
ストックの登録画面ですが、現状ストックに登録した数が増えてきておりまた登録画面内での並び替えもできないため、登録時にいちいちブラウザのサーチをかける必要が出ています（つまり記事内に同じキーワードがあるとそれも検索されることになり、導線がかなり長くなる）。ストック登録のポップアップ画面上でインクリメンタルサーチができるように、インクリメンタルサーチに対応した検索窓があると良いと思うのですが、いかがでしょうか？

**Comment by tanaka-asami415** [comment]
@Teruichi81 
ご意見ありがとうございます！🙏
ストック周りの体験改善の際に、合わせて検討させていただきます！

**Comment by kimkim0814** [comment]
@Teruichi81
大変、お待たせしました！

ストック時に検索できるようになりました✨
https://qiita.com/release-notes/2026-05-12

ご要望をお寄せいただきありがとうございました🙏
また、ストック機能は他にも色々使いやすくアップデートしているので、ぜひご活用ください！
https://blog.qiita.com/release-stock-share/
